packages/react-atlas-core/src/index.js

```diff
@@ -18,6 +18,7 @@ export { GridRowCore } from "./gridRow";
 // export RadioGroup from "./radio";
 export { SnackbarCore } from "./snackbar";
 export { TableCore } from "./table";
+export { TableHeaderColumnCore } from "./tableHeaderColumn";
```

Comment:
Maybe we should call TableHeaderColumnCore something shorter like TableHeaderCore or THeaderCore, or something so user's don't have to type as much.